dev/NumberBox/InteractionTests/NumberBoxTests.cs

```diff
@@ -316,14 +316,14 @@ public void ScrollTest()
 
                 FindTextBox(numBox).SetFocus();
```

Comment:
Marcel, can you try adding a WaitForIdle here after setting focus ? From the test failure in downlevel, the verify with the value of 2 is failing with value of 0 which might be because the mouse wheel events are going before focus is set.

Reply:
Good idea, merged with master and added a wait after setting focus.